pages/interface/components/ContentList/index.js

```diff
@@ -147,3 +147,53 @@ export default function ContentList({ contentList: list, pagination, paginationB
     return <EmptyState title="Nenhum conteúdo encontrado" {...props} />;
   }
 }
+
+function Ad({ ad }) {
+  if (!ad) {
+    return;
+  }
+
+  const link = ad.source_url || `/${ad.owner_username}/${ad.slug}`;
+  const isAdToExternalLink = isExternalLink(link);
+  const domain = isAdToExternalLink ? `(${getDomain(link)})` : '';
+  const title = ad.title.length > 70 ? `${ad.title.slice(0, 70)}...` : ad.title;
```

Comment:
Se for mudar mais alguma coisa no código. Tenta isto:

```suggestion
  const title = ad.title.length > 70 
      ? ad.title.substring(0, 67).trim().concat('...')
      : ad.title;
```